Add semester 1 newsletter (#727)
CSSUoB/cssuob.github.io: _collections/_news/2024-12-14-2024-25s-semester-1-overview.md

---
layout: post
published: true
title: CSS' Best Semester - 2024/25 Semester 1 Overview
thumbnail: /assets/images/2024-semester-1/thumbnail.png
header-image: /assets/images/2024-semester-1/banner.png
---
What a blazing start it's been for CSS - there's so much that's happened, even I've struggled to keep up! 

If you need a quick reminder on what's happened, let's go over everything...

# What's been on this semester

> Since our welcome week, we've been running a series of new events mixed in with old ones, both with old friends and new
> ones. We kicked things off with our classic Cream Tea event, joined by TeaSoc! Even if we ended up buying Sainsbury's
> full stock of scones and forgot the cream, it was still an incredibly successful event, giving us the chance to catch
> up with everyone after the chaos of welcome week. Thank you once again to TeaSoc for hosting this event with us <3

Reviewer: ```suggestion
Since our welcome week, we've been running a mix of new and recurring events, kicking off first with Cream Tea with TeaSoc! 
Even if we ended up buying Sainsbury's full stock of scones and forgot the cream, it was still an incredibly successful 
event, giving us the chance to catch up with everyone after the chaos of welcome week. Thank you once again to
TeaSoc for hosting this event with us <3
```

![A view of the room for the Cream Tea event.](/assets/images/2024-semester-1/cream-tea.png)

On the more technical side, we've been able to host a few workshops held by fellow students on what interests them -
the first two held by Amrit, who bathed us in his love for Lua (the presentation *itself* being written in Lua!), and 
Matt, who travelled across the country to come back to UoB and give a beginner's workshop on Python! 

![An image of one of Amrit's slides for his Lua Workshop, which contains an image of him with a Lua cap, a Lua shirt, a
Lua mug and a laptop with Lua on it.](/assets/images/2024-semester-1/lua-workshop.png)

> After that, we leaned back into our social side and ran our annual Halloween Quiz, where we saw a vast range of bizarre
> costumes and teams battling for wit to win a swarm of Blahajs, and an Aftonsparv for the costume competition! Despite
> a marking error at the end that did end up affecting the final scores (oops), the event went wonderfully and it was 
> incredible to see so many people dressed up.

Reviewer: ```suggestion
After that, we leaned back into our social side and ran our annual Halloween Quiz, where we saw a vast range of bizarre
costumes and teams battling to win a swarm of Blahajs, and an Aftonsparv for the costume competition! Despite
a marking error at the end that did end up affecting the final scores (oops), the event went wonderfully and it was 
incredible to see so many people dressed up.
```

![A picture of the Halloween Quiz inside of the Computer Science atrium.](/assets/images/2024-semester-1/halloween-quiz.png)

> Then, sometime in between, we got declared war upon by Bad Film Society, where they egregiously accused us of using their
> "brand identity" for some harmless publicity... needless to say, I will break my silence on this matter and will confirm
> that I have not tolerated this baseless slander, and that we have spoken diplomatically to find a negotiation that will 
> appease both parties, but I fear that we will have to resort to measures that will take us both down, as I will not 
> kneel to Bad Film's unreasonable demands and let them step over CSS like a doormat. We will win this war, one way or 
> another!! 🔥🔥🔥

Reviewer: ```suggestion
Then, sometime in between we were declared war upon by Bad Film Society, who egregiously accused us of using their
"brand identity" for some harmless publicity... needless to say, I will break my silence on this matter and will confirm
that I have not tolerated this baseless slander, and that we have spoken diplomatically to find a negotiation that will 
appease both parties. However, I fear that we will have to resort to measures that will take us both down, as I will not 
kneel to Bad Film's unreasonable demands and let them step over CSS like a doormat. We will win this war, one way or 
another!! 🔥🔥🔥
```

(For anyone struggling to read the post, you can see it in full here: <https://www.instagram.com/p/DCgljIPtNO2/> )

![A screenshot of Bad Film's declaration of war.](/assets/images/2024-semester-1/bad-film-war.png)

Towards the end of the semester, we rounded off with three big events - this being the Bar Crawl, our usual Christmas Meal
and the return of the legendary Discord Botathon!

> On the 30th of November, we hosted a bar crawl that traversed throughout the city of Birmingham, starting at Mr Mulligan's,
> making our way to the energetic Heidi's, partying our way to the Brasshouse before finally finishing off at the Soloman
> Cutler, concluding an incredibly successful bar crawl.

Reviewer: ```suggestion
On the 30th of November, we hosted a bar crawl that traversed throughout the city of Birmingham, starting at Mr Mulligan's,
making our way to the energetic Heidi's, partying our way to the Brasshouse before finally finishing off at the Soloman
Cutler.
```

![A picture of members at the bar crawl inside the Soloman Cutler.](/assets/images/2024-semester-1/bar-crawl.png)

Following that, on the 5th of December, we held our Christmas Meal, returning once again to the Green Man in Harborne!
We just happened to visit on a quiz night as well, so some groups of CSS members formed teams and participated in the
quiz - one getting second place, and another getting second to last. Regardless, it was a wonderful way to conclude the
semester, and it was wonderful to see so many people there!

![A picture of members at the Christmas Meal in the Green Man, some students sitting at tables.](/assets/images/2024-semester-1/christmas-meal.png)

> And lastly, on the 8th of December, we were able to host the Discord Botathon - the last iteration being held in 2021.
> Seven different teams responded to the call and built a Discord bot from scratch, learning from the bot workshop held 
> the day beforehand. Here's how the final standings looked:

Reviewer: ```suggestion
Finally, on the 8th of December we hosted the Discord Botathon, the first since 2021.
Seven different teams responded to the call and built a Discord bot from scratch, some learning from the bot workshop held 
the day before. Here's how the final standings looked:
```

to CSS and what we do early on, then allow people to settle into the society in their own time and if they choose to do
so; and that is still true today. However, it's still left us on committee lost for words, and I think it speaks for the
effort that they've all put in this semester. Thus, I want to give my most sincere thank you to the entirety of CSS
committee this semester for making the impossible possible, putting everything they have into what they love, and being
an unshakable team that can withstand even the toughest of tasks.

But of course, let's not forget the reason we're actually in this position - it's thanks to you. CSS has maintained its
unique and eccentric community throughout the years, but it wouldn't be that way without the members that make up that
community. Thus, whether you've given a workshop, joined us in events or simply hanged around in Discord, CSS would not 
be the same without you - thank you for making this our best semester in CSS' history. 💙

# What's next
As I round off this newsletter, I'd like to give you a brief glimpse into what we're looking to prepare next semester. 

> We've already started planning and even finalisation of our larger events, such as our trip to the National Museum of
> Computing, the glamorous ball, and the AGM to elect the entirety of committee for the new year. However, we also have a
> mystery event that we are currently working with the school on, as well as a few other societies to participate in. It's
> been a long while since we've seen this kind of event at UoB, but perhaps some events we've run this semester and will
> run the next semester too may give you a small idea... oh yeah, and calling for peace in the chaos of war against Bad
> Film.

Reviewer: ```suggestion
We've already started planning our larger events, such as our trip to the National Museum of
Computing, the glamorous Ball, and the AGM to elect the entirety of committee for the new year. However, we also have a
mystery event that we are currently working with the school on, as well as a few other societies to participate in. It's
been a long while since we've seen this kind of event at UoB, but perhaps some events we've run this semester and will
run the next semester too may give you a small idea... oh yeah, and calling for peace in the chaos of war against Bad
Film.
```
